DOC-532: update Enterprise overview, editions and quickstarts (#1784)
hazelcast/hz-docs: docs/modules/getting-started/pages/get-started-enterprise.adoc

= Get started with Hazelcast {enterprise-product-name}

> :description: This quickstart will get you started with Hazelcast Platform {enterprise-product-name} and take you from installation all the way through to a working three member cluster with persistence enabled.

Reviewer: ```suggestion
:description: This tutorial will get you started with Hazelcast Platform {enterprise-product-name} and take you from installation all the way through to a working three member cluster with persistence enabled.
```

:page-enterprise: true
:page-aliases: enterprise-overview.adoc, persistence.adoc

{description}

> == What you'll learn

Reviewer: Our template uses Overview etc. headings. Your call though...

In this tutorial, you’ll do the following:

> * Install Hazelcast Enterprise with a trial license

Reviewer: add full stops to end of list items

* Create a cluster with three members with persistence enabled
* Create a map and view the data in Management Center
* Restart the cluster and check the data persists

The tutorial should take approximately 20 minutes to complete.

> // get feedback on timing

Reviewer: about 2 days probably if you ask me to do it :-)

// Some examples do same IP address but different ports - does this also work?
<2> Replace the `<your_license_key>` placeholder with the Hazelcast {enterprise-product-name} license key from step 2.
<3> By default, Hazelcast creates the `/opt/hazelcast/persistence` directory to store the persisted data.
However, using a Docker image, you cannot create any files inside the container. This line of the command mounts a directory
on your local drive to the container, which in this case is `persist` under the home directory. This way, the container will use this local
directory to save the persisted data. You need to define a separate persist mapping for each member e.g. the second member example below uses `~/persist2`.
// more info / verification of this section
<4> The first port number should match the port assigned in <1>. For subsequent members (e.g. second, third), you should increment the first port number but keep the second port number i.e. 5702:5701, 5703:5701.

> 

Reviewer: Is it worth including a tip about cutting & pasting into Notepad or similar so you can update license key etc. or is this too noddy for our audience?

('3', 'Jane');
----
. Exit the SQL shell using the `exit` command.

== Step 7: View the map in Management Center

. Open another *new* terminal and start Management Center using the following command:
+
[source,shell,subs="attributes+"]
----
docker run \
    --network hazelcast-network \
    -p 8080:8080 hazelcast/management-center:{page-latest-supported-mc}
----
. In a web browser, go to `localhost:8080` and enable Dev Mode.
// +
// remove as unnecessary image:mc-dev-mode.png[Enabling Dev mode in Management Center]
. You should now see a *Cluster Connections* screen. 
.. Click *Connect Cluster*.
.. Enter a *Cluster Name* of `hello-world`.
.. Enter the IP addresses and ports of the three members in *Member Addresses*, using commas to separate entries.
// need to check - example image shows same IP but different ports yet instructions are separate IP and ports
.. Click *Connect* to connect to the cluster.
+
image:connect-cluster.png[Connecting Management Center to the cluster]
. You may have to wait 10 seconds or so for the cluster to show as active and with three members.
+
image:cluster-connected.png[Management Center is now connected to the cluster]
// resize image

> . Click *View Cluster* and go to **Storage > Maps**. You should be able to see the map you created has data with three entries. If you want to view the details for `mydistributedmap`, enable the *Map Statistics* box.

Reviewer: ```suggestion
. Click *View Cluster* and go to **Storage > Maps**. You will see that the map you created has data with three entries. If you want to view the details for `mydistributedmap`, enable the *Map Statistics* box.
```

+
image:cluster-maps.png[Map listing]
// image:map-details.png[Map details]

== Step 8: Restart the cluster

. Go to **Cluster** > **Administration**, and select the **Cluster State** tab.
. Click **Shutdown** and confirm the dialog that appears.
+

> Management Center should now show it is disconnected from the cluster, which you can confirm by checking your terminal windows, which should show the members exited to shell and similar output to:

Reviewer: ```suggestion
Management Center now shows it is disconnected from the cluster, which you can confirm by checking your terminal windows, which will show the members exited to shell and similar output to:
```

+
[source,shell,subs="attributes+"]
----
2025-07-18 15:21:50,538 [ INFO] [hz.eloquent_cannon..clusterShutdown] [c.h.i.i.Node]: [host.docker.internal]:5702 [hello-world] [5.5.6] Hazelcast Shutdown is completed in 124 ms.
----
+

> . Restart the cluster by starting the members up again by rerunning the commands in steps 4 and 5 above

Reviewer: full stops at the end of the steps?

> . When the members have restarted you can view your map data in Management Center and confirm the entries still exist. If persistence was not enabled, the data would be lost after cluster shutdown. 

Reviewer: ```suggestion
. When the members have restarted, you can view your map data in Management Center and confirm the entries still exist. If persistence was not enabled, the data would be lost after cluster shutdown. 
```

== Step 9: Clean up

> . Repeat the steps above to shutdown the cluster 

Reviewer: ```suggestion
. Repeat the steps above to shut down the cluster 
```
